src/NUnitFramework/tests/Attributes/SingleThreadedFixtureTests.cs

```diff
@@ -40,10 +40,10 @@ public void TestRunsOnSameThreadAsParent()
             Assert.That(Thread.CurrentThread, Is.EqualTo(ParentThread));
         }
 
-        [Test]
-        public void TestWithTimeoutIsInvalid()
+        [Test, Timeout(100)]
+        public void TestWithTimeoutIsValid()
         {
-            CheckTestIsInvalid<SingleThreadedFixture_TestWithTimeout>("TimeoutAttribute may not be specified");
+            Assert.That(Thread.CurrentThread, Is.EqualTo(ParentThread));
```

Comment:
Correction - they are not the same because one is in a SingleThreaded fixture and tests that it is now legal to have a timeout in such a fixture. It wasn't before.